### ISSUE TITLE
Setting the size of the Croppr instance itself

### ISSUE THREAD

**Issue by sahandnayebaziz** [jamesssooi/Croppr.js#2]
I am very excited to see your library! I'm just running into one issue trying to set it up. I'm trying to instantiate Croppr on an image, and though the image has css on it to make it fit on the screen, the Croppr instance displays a little bit bigger. (Looks like maybe it displays the image at its original size?)

Is there any way to set the size of the Croppr view when the image itself is bigger than the screen or the div you want to use it in?

Here is a screenshot to show you what I'm seeing. Before instantiating Croppr, the image is fit into the div. After instantiating Croppr, the image is overflowing in the div.

<img width="634" alt="screen shot 2017-04-24 at 4 03 28 pm" src="https://cloud.githubusercontent.com/assets/909340/25362385/784553e8-2908-11e7-8388-d9368dac9d05.png">

Awesome library in general, by the way :)

**Comment by sahandnayebaziz** [comment]
p.s. I tried changing the CSS of one of Croppr's divs, but was unsure how to "refresh" Croppr, so to speak, to let it know that the container size has changed. Not sure if that's how you'd recommend it to happen, anyway :)

**Comment by sahandnayebaziz** [comment]
A faster way to replicate the situation I'm talking about is by changing the unsplash line in `index.html` to something like...

```html
<img src="https://unsplash.it/2000/2000/?random" alt="" id="cropper">
```

Does handling images that are big (like images users upload for profile pictures, for example) seem feasible to you, or potentially out of scope?

**Comment by jamesssooi** [comment]
For Croppr to work on scaled images, the proper way is to resize *all* its `img` tags and not the `div` tags. For instance:

```css
.croppr img {
  width: 500px;
}
```

But I understand that it does seem like a hacky, undocumented way to get it to work. I suppose a simple fix would be for Croppr to simply retain existing styling and class names of the `img` tag after initialising (it currently does not).

**Comment by sahandnayebaziz** [comment]
I see. I'll try this! Another way to deal with this, that I came to yesterday, was to resize the image first. I've added a bit more code to handle that part, which is fine, but I do feel like cropping images straight from a users filesystem that might already be very large might be a common use case.

I'll leave it to you to think about. Not sure where to go with it next. Thank you @jamesssooi :)
